front/src/applications/editor/tools/routeEdition/components/RouteMetadata.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { RouteEntity } from 'types';
+
+export const RouteMetadata: FC<{
+  entity: RouteEntity;
+  disabled: boolean;
+  disableTransit: boolean;
+  onChange: (e: boolean) => void;
+}> = ({ entity, onChange, disableTransit, disabled }) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="form-group" style={{ opacity: disabled ? 0.5 : 1 }}>
+      <div className="row">
+        <span className="col-sm-6 col-form-label">
+          {t('Editor.tools.routes-edition.start_direction')}
+        </span>
+        <div className="col-sm-6">
+          {!disabled && (
+            <span>
+              {t(
+                `Editor.tools.routes-edition.directions.${entity.properties.entry_point_direction}`
+              )}
+            </span>
+          )}
+        </div>
+      </div>
+      <div className="row">
+        <label className="col-sm-6 col-form-label" htmlFor="include-release-detectors">
+          {t('Editor.tools.routes-edition.include-release-detectors')}
+        </label>
+        <div className="col-sm-6">
+          <input
+            type="checkbox"
+            id="include-release-detectors"
+            checked={disabled ? undefined : !!entity.properties.release_detectors.length}
```

Comment:
When selecting a candidate for the first time, we have this warning in the console

![Capture d’écran 2024-01-18 à 17 20 17](https://github.com/osrd-project/osrd/assets/113104137/a9b3adf8-7edd-4e29-afa7-6821cdeaaad8)

Is it necessary that the checkbox is at undefined if disabled ?

```suggestion
            checked={!!entity.properties.release_detectors.length}
```

Reply:
It's just to indicate to the user that there is no data. 
So when a user select a route, there is a visual diff (in addition of the opacity ).